bootstrap/bpdoc/bpdoc_test.go

```diff
@@ -44,3 +44,34 @@ func TestNestedPropertyStructs(t *testing.T) {
 		}
 	}
 }
+
+func TestAllPackages(t *testing.T) {
+	packages, err := AllPackages(pkgFiles, moduleTypeNameFactories, moduleTypeNamePropertyStructs)
+	if err != nil {
+		t.Fatalf("expected no errors, got %s", err)
+	}
+
+	if numPackages := len(packages); numPackages != 1 {
+		t.Fatalf("Expected 1 package, got %q", numPackages)
```

Comment:
Wouldn't it be more helpful if this prints out the actual contents of packages when it fails?

Reply:
Done.